src/__tests__/analysisStore.test.ts

```
@@ -0,0 +1,978 @@
+import type { 
+  CompleteAnalysis, 
+  AnalysisProgress, 
+  AnalysisHistoryEntry,
+  FilmRole 
+} from '@/types/analysis';
+import { ChatMessage } from '@/types/ChatMessage';
+
+// Mock crypto.randomUUID if not available
+Object.defineProperty(global, 'crypto', {
+  value: {
+    randomUUID: () => 'test-uuid-' + Math.random().toString(36).substr(2, 9)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)